queries/highlights.scm

```diff
@@ -109,11 +109,4 @@
   (imaginary_literal)
 ] @number
 
-[
-  (true)
-  (false)
-  (nil)
-  (iota)
-] @constant.builtin
```

Comment:
I think we should preserve the syntax highlighting of these constructs by doing string matching on identifiers, using code like this JavaScript example:

https://github.com/tree-sitter/tree-sitter-javascript/blob/master/queries/highlights.scm#L15-L21